doc/Requirements Specification/Parts/Non-Functional Requirements.lyx

```diff
@@ -191,6 +191,41 @@ name "/Q30/"
 
 \begin_layout Labeling
 \labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/Q40/"
+
+\end_inset
+
+ Beagle needs a define time to run in dependencey of the 
```

Comment:
Oh, are we talking about the *timeout* here?

If yes, adapting the timeout to previous runs is a great idea!

I thought you’d like to specify how long Beagle is allowed to run to finish a measurement.